scripts/package.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+set -e
+
+BASE_TAG=""
+LICENSE_FILE="./License"
+PACKAGES=()
+FINAL_TAG=""
+REGISTRY="registry.licensing.aica.tech"
+LOCAL_IMAGES=no
+BASE_IMAGE="base"
+
+usage() {
+  if [ -n "$1" ]; then
+    echo "Error: $1"
+  fi
+  echo "Usage: $0 [options]"
+  echo "Options:"
+  echo "  --base <version>               base image version"
+  echo "  --packages <pkg1> <pkg2>       packages to install (e.g. 'collections/ur' or 'components/gear-assembly')"
+  echo "  --license <file>               license file to use to connect to the registry (default: '${LICENSE_FILE}')"
+  echo "  --tag <tag>                    docker tag to use for the final image"
+  echo "  -h, --help                     display this help"
+  if [ -n "${DEBUG_CLI}" ]; then
```

Comment:
DEBUG_CLI is never set, whats the intended behavior here?

Reply:
It's to be used by developers like us, by adding `DEBUG_CLI=anything` before calling the tool (you can export it too if you prefer).